src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/staves/StormStaff.java

```diff
@@ -106,7 +107,7 @@ private void useItem(Player p, ItemStack item, Location loc) {
     }
 
     @ParametersAreNonnullByDefault
```

Comment:
this can be removed now, right?

Reply:
Nah, shouldn't have added nullable to that item, there are usage right below it